src/index.html

```diff
@@ -9,6 +9,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__block">
+        <img
+          src="./images/logo.png"
+          alt="company logo"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item first-child">
+            <a href="#" class="is-active">Apple</a>
```

Comment:
It's better to use more accurate href attributes for nav links. Fix it everywhere  
```suggestion
            <a href="#Apple" class="is-active">Apple</a>
```